rust/foxglove/src/protobuf.rs

```diff
@@ -151,6 +172,10 @@ impl ProtobufField for u8 {
     fn write(&self, buf: &mut impl bytes::BufMut) {
         encode_varint((*self).into(), buf);
     }
+
+    fn encoded_len(&self) -> usize {
+        prost::encoding::encoded_len_varint(*self as u64)
```

Comment:
Is a u8 ever written as a varint?

Reply:
Yes, and anything above 127 (highest bit set) will use two bytes. I added a test.